tests/integration/expected_test_results/page_crawler/crawler-access-product-page.html

```diff
@@ -515,6 +515,50 @@ <h1 class="title-3" property="food:name" itemprop="name">Only-Product – Nutell
             <div class="card" style="border-top:none">
                 <div id="prodBanner" style="display:none"></div>
                 <div class="card-section">
+                    
```

Comment:
Why has this test result changed? It looks like it added code for an image that does not exist: 

`<img style="border-radius:10px;margin-bottom:0.5rem;" class="product_image" src="" width="" height="" alt="" itemprop="thumbnail" loading="lazy">`